charts/invenio/values.yaml

```diff
@@ -162,21 +162,28 @@ persistence:
 
 redis:
   enabled: true
-  image: "redis:7.2.1"
-  host: ""
-  persistence:
-    enabled: true
-    name: "redis-volume"
-    access_mode: ReadWriteMany
-    size: 5G
-    storage_class: ""
-  resources:
-    limits:
-      cpu: "1"
-      memory: 2Gi
-    requests:
-      cpu: 500m
-      memory: 500Mi
+  auth:
+    enabled: false  # Dangerous! This lets Invenio connect to Redis unauthenticated!
+  master:
+    disableCommands: []  # Dangerous! This lets us run the `FLUSHALL` and `FLUSHDB` commands! Unfortunately, they are required by the wipe_recreate.sh script when installing Invenio.
+    resources:
+      limits:
+        cpu: "1"
+        memory: 2Gi
+      requests:
+        cpu: 500m
+        memory: 500Mi
+  replica:
+    disableCommands: []  # Dangerous! This lets us run the `FLUSHALL` and `FLUSHDB` commands! Unfortunately, they are required by the wipe_recreate.sh script when installing Invenio.
+    resources:
+      limits:
+        cpu: "1"
+        memory: 2Gi
+      requests:
+        cpu: 500m
+        memory: 500Mi
+
+redisExternal: {}
```

Comment:
I saw @Samk13 talking about this somewhere, but in any case: currently the way to use an external Redis would be to just set `redis.enabled: false` and then "manually" set all the relevant InvenioRDM environment variables (e.g. `RATELIMIT_STORAGE_URL = "redis://cache:6379/1"`, etc.). 

I'm not sure if that's a good idea in general since one might forget to update a config variable and find out later that a Redis-related InvenioRDM feature doesn't work (e.g. caching or rate-limiting). I assume fixing this at its root would mean using `(invenio.)redis.host_name` to build the default configs... We could maybe split the configuration part into a separate task, since this has to be done for all services replaced with dependencies/subcharts.

Reply:
> currently the way to use an external Redis would be to just set redis.enabled: false and then "manually" set all the relevant InvenioRDM environment variables 

Sure, but how does one manually do that? That's basically what I want the `redisExternal` for. See my other "replace-*" PR's and say if you think it makes sense; I happened to implement the logic in full in those PRs.

Reply:
I'm sorry, just checked and I'm wrong, this "auto-injection" [does happen actually already](https://github.com/inveniosoftware/helm-invenio/blob/528369e2ca380d72dfbaf79394b2b4689431dc5c/charts/invenio/templates/invenio-configmap.yaml#L7-L13). 

The only issue I see is when one would need authentication for an external Redis host. I see in the other PRs that there's sometimes a secret involved or a "raw password" value in `values.yml`. I think this requires some discussion to align on which solution we go for, but for simple cases like here in Redis, where auth is not required internally, we can just accept and merge things as is.

Reply:
Alright. Then I'll remove `redisExternal` until we decide on how it should be incorporated.